docs/cel2/whats-changed/l1-l2.md

```diff
@@ -77,8 +75,12 @@ Old blocks, transactions, receipts and logs will still be accessible via the RPC
 
 In general the changes involve additional extra unset fields that have been added upstream but were not present on historical Celo L1 objects, and the removal of some unnecessarily set fields on Celo L1 objects.
 
-For in depth details of what is changed see here - (TODO add section in specs covering this)
+For in depth details of what is changed see [here](https://specs.celo.org/l2_migration.html).
+
+### Epoch block receipts
```

Comment:
@piersy Would appreciate your eyes on this section to make sure I got this right. The previous version of this section was in the op -> cel2 diff section which didn't seem right. It also read as if all block receipts were going away which I don't think is right either.

Reply:
It wasn't just epoch blocks, it was any block where system contracts emitted logs, which up until gingerbread was all blocks. So this section should be called "Block receipts" and the references to epoch blocks should be removed.